cmd/node/apis/types.go

```diff
@@ -1,14 +1,10 @@
-package main
+package apis
 
-type errResp struct {
+type HandleErrRsp struct {
```

Comment:
use errResp ? handle is verb, maybe not suitable for struct name, but for function name. 
rename to private